webrender/src/prim_store.rs

```diff
@@ -726,7 +726,7 @@ impl TextRunPrimitiveCpu {
         display_list: &BuiltDisplayList,
         frame_building_state: &mut FrameBuildingState,
     ) {
-        if !allow_subpixel_aa {
+        if !allow_subpixel_aa && self.font.bg_color.a == 0 {
```

Comment:
shouldn't this be `!= 1.0` ?

Reply:
We use the ```!= 0``` check in a few places, and the comment where the field is declared says:

```
    /// When bg_color.a is != 0 and render_mode is FontRenderMode::Subpixel,
    /// the text will be rendered with bg_color.r/g/b as an opaque estimated
    /// background color.
```

So I think it's fine to leave it as is? Perhaps I should add an assert that the alpha in this case is either 0 or 255 though, since any other value doesn't really make sense here?

Reply:
yeah, the assert sounds like a good compromise!

Reply:
OK, rebased + added an assert in ```FontInstance::new```.